scripts/database.py

```
@@ -18,33 +18,108 @@
+            set_json = os.path.join(item_path, f"{item}.json")
+            if os.path.exists(set_json):
+                with open(set_json, "r", encoding="utf-8") as f:
+                    cards.extend(json.load(f))
+    return cards
+
+
+def sync_alternate_versions(all_cards):
+    parent = {}
+
+    def find(i):
+        if parent.setdefault(i, i) == i: return i
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)